src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -1285,6 +1304,8 @@ def get_prefix_bias_name(self) -> str:
         checkpoint=_CHECKPOINT_FOR_DOC,
         output_type=TFMaskedLMOutput,
         config_class=_CONFIG_FOR_DOC,
+        expected_output="'P a r i s'",
+        expected_loss=0.81,
```

Comment:
I think it's better to change `_CHECKPOINT_FOR_DOC` from `"bert-base-cased"` to `"bert-base-uncased"`, so it matches the one defined and used in PyTorch Bert.

And the result should be the same in `Bert`: `paris` and `0.88`.

Reply:
I have changed the `_CHECKPOINT_FOR_DOC` as your suggestion